src/MagnumPlugins/AssimpImporter/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+#
+#   Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017
+#             Vladimír Vondruš <mosra@centrum.cz>
+#   Copyright © 2017 Jonathan Hale <squareys@googlemail.com>
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#   copy of this software and associated documentation files (the "Software"),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom the
+#   Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included
+#   in all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#
+
+find_package(Assimp REQUIRED)
+
+if(BUILD_STATIC)
+    set(MAGNUM_ASSIMPIMPORTER_BUILD_STATIC 1)
+endif()
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
+               ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
+
+set(AssimpImporter_SRCS
+    AssimpImporter.cpp)
+
+set(AssimpImporter_HEADERS
+    AssimpImporter.h
+    visibility.h)
+
+# Objects shared between plugin and test library
+add_library(AssimpImporterObjects OBJECT
+    ${AssimpImporter_SRCS}
+    ${AssimpImporter_HEADERS})
+target_include_directories(AssimpImporterObjects PUBLIC
+    $<TARGET_PROPERTY:Magnum::Magnum,INTERFACE_INCLUDE_DIRECTORIES>
+    ${PROJECT_SOURCE_DIR}/src
+    ${PROJECT_BINARY_DIR}/src)
+target_compile_definitions(AssimpImporterObjects PRIVATE "AssimpImporterObjects_EXPORTS")
+if(NOT BUILD_STATIC OR BUILD_STATIC_PIC)
+    set_target_properties(AssimpImporterObjects PROPERTIES POSITION_INDEPENDENT_CODE ON)
+endif()
+set_target_properties(AssimpImporterObjects PROPERTIES FOLDER "MagnumPlugins/AnyImageImporter")
+
+# AssimpImporter plugin
+add_plugin(AssimpImporter
+    "${MAGNUM_PLUGINS_IMPORTER_DEBUG_BINARY_INSTALL_DIR};${MAGNUM_PLUGINS_IMPORTER_DEBUG_LIBRARY_INSTALL_DIR}"
+    "${MAGNUM_PLUGINS_IMPORTER_RELEASE_BINARY_INSTALL_DIR};${MAGNUM_PLUGINS_IMPORTER_RELEASE_LIBRARY_INSTALL_DIR}"
+    AssimpImporter.conf
+    $<TARGET_OBJECTS:AssimpImporterObjects>
+    pluginRegistration.cpp)
+if(BUILD_STATIC_PIC)
+    set_target_properties(AssimpImporter PROPERTIES POSITION_INDEPENDENT_CODE ON)
+endif()
+target_include_directories(AssimpImporter PUBLIC ${PROJECT_SOURCE_DIR}/src)
+target_link_libraries(AssimpImporter Magnum::Magnum Assimp::Assimp AnyImageImporter)
```

Comment:
This will fix the Travis failure:

```cmake
target_link_libraries(AssimpImporter Magnum::Magnum Assimp::Assimp)
if(CORRADE_TARGET_WINDOWS)
    target_link_libraries(AssimpImporter AnyImageImporter)
endif()
```

To explain: the plugins are built as `MODULE` libraries on almost all platforms except Windows (where I had some issues *way back* and didn't have time to look into them yet). That means they *can* have unresolved symbols, because the plugin manager will handle that on runtime. So it's not needed (not even possible) to link AnyImageImporter to this.